languages.toml

```diff
@@ -3680,6 +3680,18 @@ language-servers = [
 indent = { tab-width = 2, unit = "  " }
 grammar = "javascript"
 
+[[language]]
+name = "gherkin"
+scope = "source.feature"
+file-types = ["feature"]
+comment-token = "#"
+indent = { tab-width = 2, unit = "  " }
+grammar = "gherkin"
```

Comment:
one more small thing - this is also the default so it can be removed too

```suggestion
```

Reply:
Removed